app/controllers/courses_controller.rb

```diff
@@ -3,6 +3,12 @@ class CoursesController < ApplicationController
 
   def index
     @provider = Provider.find(params[:code]).first.attributes
-    @courses = Course.where(provider_code: params[:provider_code])
+
+    @courses = Course
+      .includes(:accrediting_provider)
+      .where(provider_code: params[:provider_code])
+
+    @courses_by_accrediting_provider = @courses
```

Comment:
Do we want to call this "accredited body" in the new code?

Reply:
Sure

Reply:
This is actually very annoying to keep in my head; I'll stick to `accrediting_provider` in the code.